subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/FindBugsPlugin.java

```diff
@@ -61,6 +64,7 @@ protected String getToolName() {
 
     @Override
     protected void beforeApply() {
+        SingleMessageLogger.nagUserOfPluginReplacedWithExternalOne("FindBugs", "SpotBugs");
```

Comment:
❌ Should use the id's of the plugins so users can find them easier.

Reply:
Same for JDepend, I assume, right?

Reply:
Right.